salt/metalk8s/metalk8s-ui/files/metalk8s-ui_deployment.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    k8s-app: ui
+    kubernetes.io/name: "MetalK8sUI"
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 1
+  selector:
+    matchLabels:
+      k8s-app: ui
+  template:
+    metadata:
+      labels:
+        k8s-app: ui
+    spec:
+      tolerations:
+        - key: "node-role.kubernetes.io/master"
+          operator: "Exists"
+          effect: "NoSchedule"
+      nodeSelector:
+        beta.kubernetes.io/os: linux
+      containers:
+      - name: metalk8s-ui
+        image: localhost:5000/metalk8s-2.0/metalk8s-ui:0.2
+        imagePullPolicy: IfNotPresent
+        resources:
+          limits:
+            memory: 500Mi
+          requests:
+            cpu: 100m
+            memory: 70Mi
+        ports:
+        - containerPort: 80
+          name: nginx-ui
+          protocol: TCP
+        livenessProbe:
+          httpGet:
+            path: /
+            port: 80
```

Comment:
`http` (if the port is renamed as suggested above).

Reply:
I'm not sure what you mean here.

Reply:
after [renaming](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#use-a-named-port) the port up in 
```
            memory: 70Mi
        ports:
        - containerPort: 80
          name: http
```
you can use the string `http` instead of the 80

Reply:
Oh ok thx